packages/data/src/plugins/persistence/index.js

```diff
@@ -223,75 +223,60 @@ function persistencePlugin( registry, pluginOptions ) {
 }
 
 /**
- * Deprecated: Remove this function and the code in WordPress Core that calls
- * it once WordPress 5.4 is released.
+ * Move the 'features' object in local storage from the sourceStoreName to the
+ * interface store.
+ *
+ * @param {*} persistence
+ * @param {*} sourceStoreName
  */
-
-persistencePlugin.__unstableMigrate = ( pluginOptions ) => {
-	const persistence = createPersistenceInterface( pluginOptions );
-
+export function migrateFeaturePreferencesToInterfaceStore(
+	persistence,
+	sourceStoreName
+) {
+	const interfaceStoreName = 'core/interface';
 	const state = persistence.get();
+	const sourcePreferences = state[ sourceStoreName ]?.preferences;
+	const sourceFeatures = sourcePreferences?.features;
+
+	if ( sourceFeatures ) {
+		const targetFeatures =
+			state[ interfaceStoreName ]?.preferences?.features;
+
+		if ( ! targetFeatures ) {
```

Comment:
Hmm, yeah, it might become a problem as we migrate other editors, I think it should maybe be something like:

```js
if ( ! targetFeatures[ sourceStoreName ] ) {
```

 I'll double check this and write some unit tests for this case, thanks for catching it.

Reply:
Yep, it was an issue, fixed and added tests in #2e57fa9